fissix/tests/test_fixers.py

```diff
@@ -3158,6 +3158,10 @@ def test_map_trailers(self):
         a = """x = list(map(f, 'abc'))[0][1]"""
         self.check(b, a)
 
+        b = """x = set(map(f, 'abc')[0])"""
+        a = """x = set(list(map(f, 'abc'))[0])"""
```

Comment:
@jreese this now fails as expected, a bug here? https://github.com/jreese/fissix/blob/master/fissix/fixer_util.py#L249-L265

I'm not sure how to fix the pattern though